### PR TITLE
LOC 2313 2531

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Updates

### CODE REVIEW

File: 2001-3000/LOC2313DabraM.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="place" xml:lang="en" xml:id="LOC2313DabraM">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Dabra Māryām</title>
+            <title>Dabra Māryām Qʷaḥayn</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply 2001-3000/LOC2313DabraM.xml:7]: This one is DM in Akkala Guzay

File: 2001-3000/LOC2313DabraM.xml [hunk 2]
```diff
@@ -49,14 +49,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
          <listPlace>
             <place type="monastery">
-               <placeName xml:lang="gez" xml:id="n1">ደብረ፡ ማርያም፡</placeName>
-               <placeName xml:lang="gez" corresp="#n1" type="normalized">Dabra Māryām</placeName>
-               
-               
-            
+               <placeName xml:lang="gez" xml:id="n1">ደብረ፡ ማርያም፡ ቈሐይን፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Dabra Māryām Qʷaḥayn</placeName>
+
```
Comment (eu-genia) [inline 2001-3000/LOC2313DabraM.xml:54]: take Qwahayn out here, update 2317
check how we encode geographic position/location, e.g. , for 2313 it should be
```
 <country ref="LOC3010Ethiop"/>
               <region notBefore="1991" type="RegionalState" ref="LOC5932Tegray"/>               
               <settlement ref="LOC1298Akkala" type="warada"/>
```
for 2317 accordingly
```
 <country ref="LOC2991Eritre"/>
               <region ref="LOC5575Saraye"/>      
<settlement ref="LOC5271Qwahay"/>         
               
```

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 2001-3000/LOC2531DaqqiI.xml [hunk 3]
```diff
@@ -48,11 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPlace>
             <place type="monastery">
-               <placeName>Däqqi Ita</placeName>
-               <placeName type="alt">Märagwǝz</placeName>
-               <placeName type="alt">Däbrä Däqqi Ita</placeName>
+               <placeName xml:lang="ti" xml:id="n1">ደቂ፡ ኢታ፡</placeName>
+               <placeName xml:lang="ti" corresp="#n1" type="normalized">Daqqi ʾItā</placeName>
+               <placeName type="alt">Marāgʷǝz</placeName>
```
Comment (eu-genia) [inline 2001-3000/LOC2531DaqqiI.xml:53]: ideally, all name variants should have the line in tgn + transcription

File: 2001-3000/LOC2531DaqqiI.xml [hunk 4]
```diff
@@ -48,11 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               <placeName type="alt">Dabra Daqqi ʾItā</placeName>
+               <placeName type="alt">Gadam Daqqi ʾItā</placeName>
             </place>
```
Comment (eu-genia) [inline 2001-3000/LOC2531DaqqiI.xml:56]: add 
```
<country ref="LOC2991Eritre"/>
               <region ref="..."/>      
<settlement ref="..."/>
```

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 2001-3000/LOC2531DaqqiI.xml [hunk 5]
```diff
@@ -48,11 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPlace>
             <place type="monastery">
```
Comment (eu-genia) [inline 2001-3000/LOC2531DaqqiI.xml:50]: `<place type="monastery gadam">`

File: 2001-3000/LOC2313DabraM.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply 2001-3000/LOC2313DabraM.xml:7]: Indeed. Difficult to see, only Dabra Maryam does not say much.

File: 2001-3000/LOC2313DabraM.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LOC2313DabraM.xml:54]: ok

File: 2001-3000/LOC2531DaqqiI.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LOC2531DaqqiI.xml:56]: Actually, I do not know why Marāgʷǝz stands here. Daqqi Ita can be/ probably is the same as  ደቂ ጣዒስ, no idea what the difference is so significant. I will add it.

File: 2001-3000/LOC2531DaqqiI.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LOC2531DaqqiI.xml:50]: But the others are also probably gadamat, should it be indicated? here the word gadam was introduced into one of the mane variants.

File: 2001-3000/LOC2531DaqqiI.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LOC2531DaqqiI.xml:56]: Ok, but not here, to Daqqi Ita.

**Review by eu-genia — COMMENTED** [review]
country/region/settlement not added

**Comment by DenisNosnitsin1970** [comment]
Everything is fixed here? If it is, please merge.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**